d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -117,6 +117,20 @@ function MFHelper() {
 
 		player = Misc.findPlayer(Config.Leader);
 	}
+	
+	
+	if (player) {
+		while (!player.area) {
```

Comment:
Consider something to not have a endless loop, try. Maybe use the small yet helpfull helper like Misc.poll

```javascript
if (!Misc.poll(() => player.area, 120*60 /* wait max 2 minutes*/, 100)) {
  throw new Error('Failed to wait for player area');
}
```
```

Reply:
fixed